R/execution.R

```diff
@@ -16,6 +16,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+#' @importFrom stats na.omit
```

Comment:
refactors to the code here were made by @ablack3 to resolve some R CDM Check issues